assets/dev/js/admin/gutenberg.js

```diff
@@ -1,10 +1,10 @@
-/* global jQuery, ElementorGutenbergSettings */
```

Comment:
Why? it's used on line 113

Reply:
its specified under `globals` in `.eslintrc.js`